packages/react-native-lockdown/src/index.js

```diff
@@ -7,17 +7,11 @@
  */
 
 /**
- * @import {SerializerConfig} from '@react-native/metro-config'
- */
-
-/**
- * @typedef {(moduleId: number | string) => string} GetRunModuleStatement
- * @see {@link https://metrobundler.dev/docs/configuration/#getrunmodulestatement}
- */
-
-/**
- * @typedef {(options: { platform?: string }) => ReadonlyArray<string>} GetPolyfills
- * @see {@link https://metrobundler.dev/docs/configuration/#getpolyfills}
+ * @import {GetRunModuleStatementFn,
+ *  GetPolyfillsFn,
+ *  LockedDownSerializerConfig,
+ *  LockdownSerializerOptions,
+ *  LockdownSerializerConfig} from './lockdown-serializer.js'
```

Comment:
manual formatting 😄

Reply:
neat 😄 

<img width="1471" alt="Screenshot 2025-06-17 at 3 03 27 pm" src="https://github.com/user-attachments/assets/871a37f8-5eb7-4191-aa0e-1496c41becdd" />

should we update the file ext and tsconfig?

(not too bothered since VS Code resolves the path fine)